Changes made to address Issues #4, #5, and #9

jillshem opened the pull request:
Progress made so far includes the following:

Contact Us Call To Action Added:
- Contact Us text is visible 
- Contact Us button is visible and clicking it leads to the Contact Us page

Contact Us page:
- 80% complete: Contact Us page has a functional contact form (with name, email, and message) 
- TODO: Contact Us page backend functionality to submit message
- Contact Us page displays social media icons link to active profiles
- NOTE: Couldn't find social media profiles

About page:
- 70% complete
- Our Mission, Our Roots
- Pillar tiles
- Host Organizations & Partners
- Why Tech4R Exists
- TODO: Our Track Record and Timelines

Pillars of Action page:
- 100% complete

okjake commented on src/app/pillars-of-action/page.tsx:
```diff
@@ -1,12 +1,30 @@
 import { Hero } from "@/components/Hero";
+import { pillars } from './pillars';
```
Love this, very clean 👍

okjake commented on yarn.lock:
Lets discuss which package manager to settle on later otherwise we'll can up with two out of sync lockfiles, might cause issues with the build

jillshem commented:
Thank you! I've removed the yarn.lock file, and am now using npm. I've also updated the classes to be a better match to the Figma design. 

I'll work on similar changes on the Contact Us and About pages next
